api/v2.0/swagger.yaml

```diff
@@ -667,6 +667,33 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
+  /p2p/preheat/instances/ping:
+    post:
+      summary: Ping status of a instance.
+      description: |
+        This endpoint checks status of a instance, the instance can be given by ID or Endpoint URL (together with credential)
+      operationId: PingInstances
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - name: instance
+          in: body
+          description: The JSON object of instance.
+          required: true
+          schema:
+            $ref: '#/definitions/Instance'
+      tags:
+        - preheat
+      responses:
+        '200':
+          description: Instance is healthy.
+        '400':
```

Comment:
Why not use the ref `$ref: '#/responses/401'`?

Reply:
I'm missed it, fix later.